js/screens/portfolio/PortfolioHomeScreen.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * CITTADINANZA DIGITALE
+ * io.italia.it
+ *
+ * @flow
+ */
+
+import * as React from 'react'
+import I18n from '../../i18n'
+import type { NavigationScreenProp, NavigationState } from 'react-navigation'
+import { Image, TouchableHighlight } from 'react-native'
+import {
+  Container,
+  Content,
+  H2,
+  Text,
+  View,
+  Left,
+  Right,
+  Button,
+  Icon
+} from 'native-base'
+import { Grid, Row } from 'react-native-easy-grid'
+import { PortfolioStyles } from '../../components/styles'
+import OperationsList from '../../components/portfolio/OperationsComponent'
+import PortfolioAPI from '../../lib/portfolio/portfolio-api'
+
+import type { Operation } from '../../lib/portfolio/types'
+import ROUTES from '../../navigation/routes'
+
+type Props = {
+  navigation: NavigationScreenProp<NavigationState>
+}
+
+/**
+ * Portfolio Home Screen
+ */
+class PortfolioHomeScreen extends React.Component<Props> {
+  static navigationOptions = {
+    title: I18n.t('portfolio.portfolio'),
+    headerBackTitle: null
+  }
+
+  constructor(props: Props) {
+    super(props)
+  }
+
+  render(): React.Node {
+    const { navigate } = this.props.navigation
+    const TITLE = I18n.t('portfolio.portfolio')
+    const cardsImage = require('../../../img/creditcards.jpg')
```

Comment:
please put this require outside the render() method which may be executed multiple times

Reply:
Gotcha